internal/config/projectconfig/init.go

```diff
@@ -1,43 +1,28 @@
 package projectconfig
 
 import (
+	"bytes"
 	"fmt"
 	"io/ioutil"
+	"os"
 	"path"
+	"text/template"
 
 	"github.com/commitdev/zero/internal/constants"
+	"github.com/commitdev/zero/internal/util"
 	"github.com/commitdev/zero/pkg/util/exit"
 	"gopkg.in/yaml.v2"
 )
 
-const exampleConfig = `name: %s
+// {{  .ShouldPushRepositories | printf "%q"}}
+const exampleConfig = `
```

Comment:
Let's give this a better name. `zeroProjectConfigTemplate`?

Reply:
Yeah, It read better 👍🏽